src/versions.rs

```diff
@@ -1,13 +1,13 @@
 /// npm: https://www.npmjs.com/package/@holochain/tryorama
-pub const TRYORAMA_VERSION: &str = "^0.16.0-dev.6";
+pub const TRYORAMA_VERSION: &str = "^0.17.0-dev.0";
 
 /// npm: https://www.npmjs.com/package/@holochain/client
-pub const HOLOCHAIN_CLIENT_VERSION: &str = "^0.17.0-dev.12";
+pub const HOLOCHAIN_CLIENT_VERSION: &str = "^0.18.0-dev.0";
 
 /// npm: https://www.npmjs.com/package/@holochain/hc-spin
-pub const HC_SPIN_VERSION: &str = "0.300.2-dev.5";
+pub const HC_SPIN_VERSION: &str = "^0.400.0-dev.0";
 
-// TODO: update to 0.3 compatible version
```

Comment:
Haha, hopefully that comes along soonish :) It's looking unlikely that 0.3 will ever be supported so many this TODO can go on the 0.3 branch?